lib/Messenger/Handler/SanityCheckHandler.php

```diff
@@ -59,12 +59,15 @@ private function process(array $jobs): void
     }
 
     /**
-     * @param PageSnippet|Asset|Concrete $element
+     * @param ElementInterface $element
      *
      * @throws \Exception
      */
     private function performSanityCheck(ElementInterface $element)
     {
+        if (!$element instanceof PageSnippet && !$element instanceof Concrete) {
```

Comment:
isn't there the chack for asset missing?

Reply:
@fashxp I add the missing Asset check. These are the only ElementTypes which have the getLatestVersion method. I thought about an Interface for this classes, but the method is on the Dao classes.

Where do you see a BC break? Shoud I remove the if and add the phpstan error to the baseline?

Reply:
I see, good point. 
The BC break idea is a more academic one ... I think, with the additional check we are good to go 👍
Thx for the efforts...